src/Expensify.js

```diff
@@ -174,6 +174,12 @@ function Expensify(props) {
             BootSplash.hide();
 
             setIsSplashShown(false);
+
+            // If the app is opened from a deep link, get the reportID (if exists) from the deep link and navigate to the chat report
+            Linking.getInitialURL().then((url) => {
+                User.setReportScreenReportID(url);
+                Report.openReportFromDeepLink(url);
+            });
```

Comment:
I am not sure why this logic is added here instead of 
https://github.com/Expensify/App/blob/214658eca48517fc3c2d087f569187ddec173ee7/src/Expensify.js#L157

Reply:
Thanks. Also should add it here